hera_mc/mc_session.py

```diff
@@ -2403,7 +2403,8 @@ def correlator_control_command(self, command, starttime=None, duration=None,
             if true, do not use anything that requires connection to correlator (implies dry run)
         """
         from . import correlator as corr
-
+        import hera_corr_cm
```

Comment:
This should be done inside the `if not dryrun` conditional.

Reply:
done